jaeger-tracerresolver/build.gradle

```diff
@@ -0,0 +1,34 @@
+description = 'TracerResolver implementation for Jaeger Tracer'
+
+dependencies {
+    compile project(':jaeger-core')
+    compile group: 'io.opentracing.contrib', name: 'opentracing-tracerresolver', version: tracerResolverVersion
+ 
+    testCompile group: 'junit', name: 'junit', version: junitVersion
+
+    signature 'org.codehaus.mojo.signature:java16:1.1@signature'
+}
+
+sourceSets {
```

Comment:
Can be this boilerplate avoided? I think it's redundant as we are sticking to default file hierarchy. 

I know this is repeated over and over it other modules, but should be used only in places where it's required to override a default conventions.